lib/ansiblelint/errors.py

```diff
@@ -1,37 +1,29 @@
 """Exceptions and error representations."""
 
 
-class Match(object):
-    """Rule violation detected during linting."""
+class Match(ValueError):
```

Comment:
I still think that it should be
```suggestion
class MatchError(ValueError):
```
https://github.com/ansible/ansible-lint/pull/884#discussion_r449854967

Reply:
done